docs/medical-api/api-reference/patient/bulk-create-patient.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: "Bulk Patient Create"
+description: "Creates multiple Patients in Metriport at once."
```

Comment:
This is only half of what the endpoint is doing, right?

Since the query is billable, I think we should make this explicit in the title and description. Perhaps:

```
title: "Bulk Patient Create and Query"
description: "Creates a batch of patients in Metriport, and initiates Document Queries for all of them."
```

Reply:
done